src/components/pages/Theme/Theme.jsx

```diff
@@ -15,6 +15,7 @@ import {
 
 // styles
 import './Theme.css';
+import '../../../styles/input.css';
```

Comment:
We need to include this in `ga.css`. That's the only file FAST is going to be referencing.

Reply:
Can we merge all style files for components to `ga.css` just before we hand it off to FAST? And for now keep them separate, so it's easier to review